receiver/statsdreceiver/README.md

```diff
@@ -21,6 +21,8 @@ The Following settings are optional:
 
 - `enable_metric_type: true`(default value is false): Enable the statsd receiver to be able to emit the metric type(gauge, counter, timer(in the future), histogram(in the future)) as a label.
 
+- `is_monotonic_counter` (default value is false): Set all counter-type metrics the statsd receiver received as monotonic.
```

Comment:
Is this true all the time? Do we need to have per counter config?

Reply:
> Do we need to have per counter config?

With the current change, it applies to all. 

I think we do need for per counter config, we could add include/exclude with regex? Although it feels more like a processor job.

Reply:
Yes. In previous PRs on this receiver I've asked that we follow the Prometheus statsd exporter's configuration, e.g.,
https://github.com/prometheus/statsd_exporter#glob-matching

I think it is fine to add this to a TODO and continue in another PR. The current PR allows a default choice, and I would say that probably the default should be true (i.e., assume monotonic by default).

Reply:
Yeah it would be great if this receiver were following the statsd exporter 👍🏼 

May I have this one merged until that come? I set the default to false since seeing this https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1789 so I don't want to cause any trouble for the DataDog exporter.